packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -475,6 +480,12 @@ export function TransferPanel() {
                   payload: false
                 })
                 setTransferring(false)
+                if (
+                  !isSmartContractWallet &&
+                  isFathomNetworkName(l2NetworkName)
+                ) {
+                  trackEvent(`Deposit ERC-20 to ${l2NetworkName} (EOA)`)
+                }
```

Comment:
I think we can get rid of the tracking code above, and just do everything here:

```suggestion
                if (isFathomNetworkName(l2NetworkName)) {
                  trackEvent(`Deposit ERC-20 to ${l2NetworkName} (${isSmartContractWallet ? 'Smart Contract' : 'EOA'})`)
                }
```

Same applies to other cases